### PR TITLE
Radfan control keep debounce

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

_Explanation of changes goes here_

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

File: Core/Src/control.c [hunk 1]
```diff
@@ -96,6 +106,7 @@ void vControl(void *params)
 
 	device_control_t radfan0 = {
 		.pdu = pdu,
+		.mc = mc,
```
Comment (jr1221) [inline Core/Src/control.c]: why?
